package-lock.json

```diff
@@ -5211,6 +5271,15 @@
         "safer-buffer": "2.1.2"
       }
     },
+    "ecurve": {
```

Comment:
This is a dependency of `bip-schnorr`, which is a dependency of `bitcore-lib`, that is the dep tree:

```
│ ├─┬ bitcore-lib@8.25.28
│ │ ├── bech32@2.0.0
│ │ ├─┬ bip-schnorr@0.6.4
│ │ │ ├── bigi@1.4.2
│ │ │ ├─┬ ecurve@1.0.6
│ │ │ │ ├── bigi@1.4.2 deduped
│ │ │ │ └── safe-buffer@5.2.0 deduped
│ │ │ ├── js-sha256@0.9.0
│ │ │ ├─┬ randombytes@2.1.0
│ │ │ │ └── safe-buffer@5.2.0 deduped
│ │ │ └── safe-buffer@5.2.1
```